Src/Plugins/SalAnalyzer/SalAnalyzer.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Targets\build.common.props" />
```

Comment:
>MSBuildThisFileDirectory [](start = 21, length = 24)

Your name doesn't conform to our existing plugins. For one thing, we use 'plugin' not analyzer. We do not suffix proejct names with analyzes.

Please call this thing 'SalUpdate' or 'SalModernization'. Pick your favorite.

Reply:
Picked.

---
In reply to: [604482931](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r604482931) [](ancestors = 604482931)